README.md

```diff
@@ -116,13 +116,6 @@ example does not link with GNU ld.bfd which is the default linker on most Linux
 installing [mold](https://github.com/rui314/mold), [lld](https://lld.llvm.org/), or GNU ld.gold
 (from GNU binutils but may be separate package) is required on Linux.
 
-On Windows and macOS vcpkg can be used by adding `-D VCPKG=ON` to the CMake configure step to
-automatically download release mode packages from GitHub Packages (this will take several minutes the first time you run CMake).
-Note that debug symbols are not built in these packages.
-
-CXX-Qt defaults to building with Qt6. If you want to build with Qt5 when both are installed,
-or you want to tell vcpkg to use Qt5, add `-D USE_QT5=ON` to the CMake configure step.
```

Comment:
This is still broadly true, just need to remove the note about vcpkg, right?

Reply:
Done in the original PR